github/issues_timeline_test.go

```diff
@@ -19,7 +19,6 @@ func TestIssuesService_ListIssueTimeline(t *testing.T) {
 
 	mux.HandleFunc("/repos/o/r/issues/1/timeline", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		testHeader(t, r, "Accept", mediaTypeTimelinePreview)
```

Comment:
Please don't delete this, but instead test for both headers.

I see that #1066 didn't test for the headers, which was my mistake as a code reviewer... so I apologize.
Please see surrounding examples of how to test for multiple headers.

This will help us to clean up and remove all preview headers when the time comes.
Thank you!